distributed/src/main/scala/asobu/distributed/service/Extractors.scala

```diff
@@ -0,0 +1,196 @@
+package asobu.distributed.service
+
+import asobu.distributed.service.Action.DistributedRequest
+import asobu.distributed.service.Extractors._
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/iHeartRadio/asobu/file/2313371925/issues/source?bid=3037259&fileBranchId=3178877#l4)